extensions/BMO/template/en/default/hook/global/header-additional_header.html.tmpl

```diff
@@ -52,3 +52,25 @@
     [% END %]
   [% END %]
 [% END %]
+
+[%# BMO - Google Analytics %]
+[% IF !user.in_group('core-security') %]
```

Comment:
This actually needs to check the insider group param.
user.in_group(Param('insidergroup'))

Reply:
Fixed